README.md

```diff
@@ -90,6 +90,8 @@ During loading, we chose to use a psycopg2 function called execute_batch which l
 
 ## Reviews ETL pipeline
 
+# TODO - talk about running nltk download
```

Comment:
Not really the right branch for this, but i'm currently working on expanding this section so it'll be replaced by this evening